src/components/user-block/react/UserBlock.tsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 
 import { Theme, Themes } from 'LumX/components';
 
-import { COMPONENT_PREFIX } from 'LumX/core/react/constants';
+import { COMPONENT_PREFIX, ORIENTATIONS } from 'LumX/core/react/constants';
```

Comment:
Shouldn't it be in the `LumX/components/index.ts` file along with  `import { Theme, Themes } from 'LumX/components';` ?

Reply:
ok done, although I think it would make more sense to get all our constants in a `constant.js` file.

Reply:
You think type definitions should be in the same file as the constants ?